README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
You are right @wkurniawan07 it's not clear who this `we` are. 

1. As you may notice, some of our orgs have `nus` in the name while others don't.  Those meant for 'NUS students only' have `nus` in it. That is because NUS employees are not allowed to use NUS brand to push products/services to third parties as it can expose NUS to legal liabilities. Also, keeping `nus` out makes them more neutral to adopters from peer universities. That is why we have not moved teammates etc. under nus-oss org. They remain independent orgs although mentioned in nus-oss org. Similarly, this org will contain workflows, conventions, tools etc we developed but available for outsiders. There is no need to link any of it to NUS just because most of us (but not all) involved in those projects are in NUS (or was in NUS at some point).

2. I don't think this repo is trying to claim as representing GitHub and we are in no position to impose this process on anyone either. We write these docs in a way that projects who follow that process can use the docs directly instead of duplicating the same content in their own docs. Others can 'take it or leave it'.  The name `oss-generic` sounds general, but it's just an identity. I wanted it to sound neutral and not connected to nus so that an outside project can say 'we follow the oss-generic process as described at {link}'.  It's like 'semantic versioning';  we don't question how come some random guys decided to name their versioning convention as 'semantic'.  

But you are right that the way it is presented can be misleading to readers. Going to revise as follows:

---

# The _OSS-Generic_ Reference Process 

The _OSS-Generic Reference Process_ described in this repo documents common practices among several OSS projects on GitHub, notably, [TEAMMATES](https://github.com/teammates/teammates), [SE-EDU projects](https://github.com/se-edu/main), [PowerPointLabs](https://github.com/powerpointlabs/powerpointlabs).

Other projects are welcome to adopt/adapt these practices if they wish to.